unified-codes/apps/web/src/components/detail/DetailViewer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { Grid, Typography } from '@unified-codes/ui'
+import { IEntity, IProperty } from '@unified-codes/data';
+
+import { IState } from '../../types';
+import { DetailSelectors } from '../../selectors';
+
+interface IEntityViewerProps {
+    classes?: { root?: string };
+    code: string;
+    description: string;
+    children?: IEntity[];
+    properties?: IProperty[];
+}
+
+type EntityViewer = React.FunctionComponent<IEntityViewerProps>;
+
+const EntityViewer: EntityViewer = ({ classes, code, description, children, properties }) => {
+    const nameField = `Name: ${description}`;
```

Comment:
Thinking here is that at some point we should probably introduce localisation (even if only in English), so these should all be constants/function calls. Think with that in mind, probably better to avoid inline strings wherever possible?

Reply:
yeah, good point. though not applicable atm - I would change when localising rather than pre-empting now.